tests/framework/extensions/clusters/dynamicSchema.go

```diff
@@ -0,0 +1,15 @@
+package clusters
+
+type DynamicSchema struct {
```

Comment:
Can you clean this up a bit Tim, that way we can add new fields if we need to so like this.

```
type Default Struct {
    StringValue

}

type SSHUser struct {
  Default
 ...
}

type ResourceFields struct {
   SSHUser
}

type Dynamic struct {
   ResourceFields 
}
```

Reply:
Sorry one last thing can you add comments? Explaining the purpose of these structs.